docs/cloud.md

```diff
@@ -45,34 +45,34 @@ Generate JWT token :
 JWT_TOKEN=$(ocm token)
 ```
 
-##### HTTP request
+### HTTP request
 
 ```bash
 JWT_TOKEN=$(curl https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token -d client_id=cloud-services -d grant_type=refresh_token -d refresh_token=${OFFLINE_TOKEN} | jq -r '.access_token')
 ```
 
-The JWT token is valid for 15 minutes and need to be provided in the header of the HTTP request.
+### API Usage example
 
 Here an example to get all the user's clusters:
 
 ```bash
-curl https://api.openshift.com/api/assisted-install/v1/clusters -H "Authorization: Bearer ${JWT_TOKEN}"
+curl https://api.openshift.com/api/assisted-install/v2/clusters -H "Authorization: Bearer ${JWT_TOKEN}"
 ```
 
-#### Agent Authentication
+## Agent Authentication
 
 Agent authentication uses a token from the Pull Secret and needs to be provided in the header of the HTTP request.
```

Comment:
Do we want to describe this more specifically?

IIRC the token is the value of a specific key in the pull secret (not the whole thing).

Reply:
definitely